docs/how-to-guides/develop-a-tool/use-file-path-as-tool-input.md

```diff
@@ -0,0 +1,98 @@
+# Use FilePath as tool input
```

Comment:
Will prefer to use `file path`

Reply:
change it to Use file path as tool input